### ISSUE TITLE
Why should FNet，and SRNet be trained respectively ？because of gradient back propagation？

### ISSUE THREAD

**Issue by hzg456** [thunil/TecoGAN#110]
Why should FNet，and SRNet be trained respectively(I mean using two  Optimizers）？
because the function `tfa.image.dense_image_warp` cannot be used for gradient back propagation？

**Comment by eewindfly** [comment]
The pro is that you can fine-tine two networks separately, however, in the source code, no different hyper-parameters are applied.
Therefore, I think respective training is not a must.
If I am wrong, please correct me.
